products/payslip_fra/response_v3/complete.json

```diff
@@ -0,0 +1,155 @@
+{
+  "api_request": {
+    "error": {},
+    "resources": [
+      "document",
+      "job"
+    ],
+    "status": "success",
+    "status_code": 200,
+    "url": "https://api.mindee.net/v1/products/mindee/payslip_fra/v3/documents/a479e3e7-6838-4e82-9a7d-99289f34ec7f"
+  },
+  "document": {
+    "id": "a479e3e7-6838-4e82-9a7d-99289f34ec7f",
+    "inference": {
+      "extras": {},
+      "finished_at": "2024-11-26T15:46:39.083000",
+      "is_rotation_applied": true,
+      "pages": [
+        {
+          "extras": {},
+          "id": 0,
+          "orientation": {
+            "value": 0
+          },
+          "prediction": {}
+        }
+      ],
+      "prediction": {
+        "bank_account_details": {
+          "bank_name": null,
+          "iban": null,
+          "swift": null
+        },
+        "employee": {
+          "address": "52 RUE DES FLEURS 33500 LIBOURNE FRANCE",
+          "date_of_birth": null,
+          "first_name": "Jean Luc",
+          "last_name": "Picard",
+          "phone_number": null,
+          "registration_number": null,
+          "social_security_number": "123456789012345"
+        },
+        "employer": {
+          "address": "1 RUE DU TONNOT 25210 DOUBS",
+          "company_id": "12345678901234",
+          "company_site": null,
+          "naf_code": "1234A",
+          "name": "DEMO COMPANY",
+          "phone_number": null,
+          "urssaf_number": null
+        },
```

Comment:
A lot of the fields are missing here, can you fill them in for the tests, please? Same thing for summary_full.rst obviously

Reply:
done